src/modules/mavlink/mavlink_receiver.cpp

```diff
@@ -797,7 +797,7 @@ MavlinkReceiver::handle_message_distance_sensor(mavlink_message_t *msg)
 	d.type = dist_sensor.type;
 	d.id = 	MAV_DISTANCE_SENSOR_LASER;
 	d.orientation = dist_sensor.orientation;
-	d.covariance = dist_sensor.covariance;
+	d.covariance = dist_sensor.covariance / (100 * 100); // cm^2 to m^2
```

Comment:
`d.covariance = dist_sensor.covariance * 1e-4f; // cm^2 to m^2`

Reply:
Isn't 100 *100 seemed for obvious?

Reply:
We rarely skip using scientific notation for this kind of data.